### ISSUE TITLE
bug, should report error when there are global classes in local report

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1215]
example,

```abap
CLASS zcl_abapgit_object_wdcc DEFINITION
  INHERITING FROM zcl_abapgit_objects_super
  PUBLIC.
ENDCLASS.
```

**Comment by larshp** [comment]
see https://github.com/abaplint/abaplint/issues/1117

**Comment by larshp** [comment]
suggest renaming https://rules.abaplint.org/superclass_final/

and including
* existence check for RAISING classes(unless they are voided), https://github.com/abaplint/abaplint/issues/1117
*

**Comment by larshp** [comment]
or have it all as part of syntax checking

**Comment by larshp** [comment]
also check rule: Class must be local(global_class)
